queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/__init__.py

```diff
@@ -49,12 +49,12 @@ def create_address(address_info: Dict, address_type: str) -> Address:
     db_address_type = address_type.replace('Address', '')
 
     address = Address(street=address_info.get('streetAddress'),
-                      street_additional=address_info.get('streetAddressAdditional'),
+                      street_additional=address_info.get('streetAddressAdditional') or '',
```

Comment:
setting  this to empty string if `None` to avoid creation of new version if the current version has empty string